14-draft.json

```diff
@@ -1132,6 +1132,49 @@
           "end"
         ]
       }
+    },
+    "membership_plans": {
+        "description": "A list of the different membership plans your hackerspace might have.",
+        "type": "array",
+        "items": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "The name of the membership plan. For example Student Membership, Normal Membership etc...",
+                    "type": "string"
+                },
+                "value": {
+                    "description": "How much does this plan cost?",
+                    "type": "number"
+                },
+                "currency": {
+                    "description": "What's the currency? It should be formatted according to <a href=\"https://en.wikipedia.org/wiki/ISO_4217\" target=\"_blank\">ISO 4217</a> short-code format.",
+                    "type": "string"
+                },
+                "renewal_interval": {
+                    "description": "How often is the membership renewed? If you select other, please specify in the description what your renewal interval is.",
+                    "type": "string",
+                    "enum": [
+                        "yearly",
+                        "monthly",
+                        "weekly",
+                        "daily",
+                        "hourly",
+                        "other"
```

Comment:
We can also merge it as is and improve it later, to not stall this PR for such a long time.